.github/workflows/ci.yaml

```diff
@@ -15,20 +19,134 @@ on:
       - cryostat-v[0-9]+.[0-9]+
 
 jobs:
-  build:
+  build-core:
+    runs-on: ubuntu-latest
+    outputs:
+      test_version: ${{ steps.get_core_version.outputs.test_version }}
+    steps:
+    - uses: actions/checkout@v4
+    - id: get_core_version
+      run: echo "test_version=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)" >> $GITHUB_OUTPUT
+    - uses: actions/setup-java@v3
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+        cache: 'maven'
+    - run: mvn -B -U clean verify
+    - uses: actions/upload-artifact@v3
+      with:
+        name: cryostat-core
+        path: /home/runner/work/cryostat-core/cryostat-core/target/cryostat-core-${{ steps.get_core_version.outputs.test_version }}.jar
```

Comment:
I think `$GITHUB_WORKSPACE` would work as a prefix here:

https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables

ie. `$GITHUB_WORKSPACE/target/cryostat-core-$version.jar`

You could probably even have `get_core_version` output the `project.build.directory` and use that instead:

```bash
$ ./mvnw -q -DforceStdout -Dexpression=project.build.directory help:evaluate
/home/work/workspace/cryostat-core/target
```

This just makes the workflow a little more generic/reusable.

Reply:
```bash
$ ./mvnw -q -DforceStdout -Dexpression=project.build.finalName help:evaluate
cryostat-core-2.25.0-SNAPSHOT
```